servers/audio_server.cpp

```diff
@@ -172,6 +172,13 @@ void AudioServer::_driver_process(int p_frames, int32_t *p_buffer) {
 
 	int todo = p_frames;
 
+	if (channel_count != get_channel_count()) {
+		// Amount of channels changed due to a device change
+		// reinitialize the buses channels and buffers
+		channel_count = get_channel_count();
```

Comment:
This line is not really needed, as you actually do it again at the start of `init_channels_and_buffers()`.

Reply:
Good catch, just fixed it.